bin/git-forgit

```diff
@@ -525,24 +525,28 @@ _forgit_reset_head() {
 
 _forgit_stash_show_preview() {
     local stash
-    stash=$(echo "$1" | cut -d: -f1)
-    _forgit_git_stash_show "$stash" | _forgit_pager diff
+    stash=$(echo "$1" | _forgit_extract_stash_name)
+    shift
+    echo "$stash" | xargs -I% git show --color=always -U"$_forgit_preview_context" % -- "$@" | _forgit_pager show
 }
 
-_forgit_git_stash_show() {
-    git stash show --color=always --ext-diff "$@"
+_forgit_stash_show_enter() {
+    local stash
+    stash=$(echo "$1" | _forgit_extract_stash_name)
+    shift
+    echo "$stash" | xargs -I% "${FORGIT}" show % "$@"
```

Comment:
Same questions regarding `xargs` and the additional arguments as noted above with `_forgit_stash_show_preview`.